src/components/PricePlan.tsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import icon from '../assets/icon-music.svg';
+
+interface Props {
+  type: 'annual' | 'monthly';
```

Comment:
You could create a type alias for 'annual' | 'monthly', considering its being used in a few places.

Reply:
I've added a new `PriceType` alias in `src/types.ts` file